v63003/outref/gtm8805.txt

```diff
@@ -0,0 +1,5 @@
+^B(1)="# Sequence of Events in Subtest"
+^B(2)="# Process 1: Process 1 currently holds ^x"
+^B(3)="# Process 2: Seized write critical section."
+^B(4)="# Process 1: Issuing a kill -9"
```

Comment:
Need one step before here indicating "Process 3: Attempting to get lock ^x:1. Should time out"